_posts/2017-01-05-graduate-devops-blog.md

```diff
@@ -0,0 +1,22 @@
+
+Whilst at Loughborough University, I was part of a sponsorship program for a defence organisation which meant I completed summer placements as well as a year in industry between my 2nd and 3rd year for that company. Giving up my summers wasn’t quite as bad as it sounds; I got paid and most importantly it gave me some invaluable experience, which has shaped my career. During my master's year at University I decided to look beyond my engineering background where I had all of my experience, and to something which would challenge me in a different way. The time I spent doing my degree and working in industry made me realise what I enjoyed doing the most was solving difficult problems and working with the latest technology in a modern fast paced environment. Sky Betting and Gaming fits this description perfectly; I could be working on IOS10 or Nougat, using UNIX or the latest MacBook Pro to create and develop a product. Sky Betting and Gaming are pushing the boundaries of what technologies the market can use, and what their employees use to fulfil the latest gaming or betting trends. 
+
+If someone told me 6 months ago I would have a good understanding of, and be able to use the likes of Ruby, Jenkins, Vim, AWS, Chef or Docker in such a short period of time I would have asked them to repeat the sentence and explain what most of those words actually meant. I don’t mean to scare anybody off, but you will need to keep an open mind when thinking about the variety of toolsets which are in existence to help you complete a job. 
+
+With a willingness to take a risk and a little step into the unknown with new toolsets or technologies you could be making yourself more efficient and also enhancing the product. This open minded approach will consequently ensure that you are constantly pushing yourself to understand different languages or previously unheard of technology, thus continuously improving yourself and the company. Keeping my mind open and taking a risk perfectly sums my transition from engineering to technology - it is the ideal platform on which to build new skills and a career. I took a risk, but it paid off. 
+
+
+Finally, take a look at the new graduate academy and if you know anyone who enjoys a challenge, then refer them to the Sky Betting and Gaming website, [Early Careers Tribe](https://skybetcareers.com/our-tribes/early-careers-tribe). Additionally read James Moorhouse Blog: [Life After Graduation](http://engineering.skybettingandgaming.com/2016/03/16/life-after-graduation/)

```

Comment:
I'd drop the link to the other blog post into a separate section of "useful links" or "related reading" or something.